ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -315,6 +315,24 @@ The description can be reverted manually if needed:
 +mgmt_interface_description: "oob_management"
 ```
 
+#### `wan_mode: autovpn` renamed `wan_mode: legacy-autovpn`
+
+With AVD version 5.0.0 the valid values for `wan_mode` key have changed. If using the `autovpn` mode in AVD 4.x, the `wan_mode` key needs tp be updated to `legacy-autovpn`.
+
+```diff
```

Comment:
It is not interesting for a user to see the schema snip. They need to see what how their vars needs to change.